MapboxNavigation/Resources/ru.lproj/Abbreviations.plist

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>abbreviations</key>
+	<dict>
+		<key>город</key>
+		<string>г.</string>
```

Comment:
@yuryleb, note that the abbreviation will appear in the case given here. So if the `name` tag in OSM contains the word “Город”, it’ll be [replaced](https://github.com/mapbox/mapbox-navigation-ios/blob/6a4754bf197d06a5368b84fdcb522db82ad5b179/MapboxNavigation/Abbreviations.swift#L37) by “г.” (I suppose the SDK could automatically match the original case, but that can be tricky because, in some languages, some abbreviations may differ in case from the original word.)

Reply:
No problem, all such Russian  "status" names should be in lower case both in full and abbreviated forms :wink: